libp2p/pubsub/pubsub_notifee.py

```diff
@@ -56,10 +60,14 @@ def __init__(
         :param network: network the connection was opened on
         :param conn: connection that was opened
         """
-        await self.dead_peers_queue.put(conn.muxed_conn.peer_id)
+        try:
+            await self.dead_peers_queue.send(conn.muxed_conn.peer_id)
+        except trio.BrokenResourceError:
+            # The receive channel is closed by Pubsub. We should do nothing here.
+            ...
 
     async def listen(self, network: INetwork, multiaddr: Multiaddr) -> None:
-        pass
+        ...
```

Comment:
Should this raise `NotImplementedError` or something?

Reply:
`PubsubNotifee.listen` just do *nothing* here. Since it is an asynchronous function, I'm going to add checkpoints here.

Reply:
Lets reserve `...` for `@abstractmethod` bodies.

I think anytime something is intentionally doing nothing it's probably good to leave a comment so the intention is clear.

Reply:
Sounds reasonable to me. Will change it.